assets/css/omise-css.css

```diff
@@ -520,3 +554,6 @@ line-height: 2;
 	margin-bottom:0px !important;
 }
 
+.terms-and-conditions-block {
+	margin-top: 10px;
+}
```

Comment:
is this only for fpx? good to use an fpx class. otherwise if it's used by other payment methods then we might have to see how this styling affects those as well